tkg/client/cluster.go

```diff
@@ -180,7 +180,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			// If `features.cluster.auto-apply-generated-clusterclass-based-configuration` feature-flag is not activated
 			// log command to use to create cluster using ClusterClass based config file and return
 			if !c.IsFeatureActivated(constants.FeatureFlagAutoApplyGeneratedClusterClassBasedConfiguration) {
-				log.Warningf("\nTo create a cluster with it, use")
+				log.Warningf("\nTo create a cluster with it, please use the following command together with customize flags like --tkr, -v, etc.")
```

Comment:
Please provide more context why these customize flags are required.

Reply:
Indeed, all customized flags are required.
When creating cluster using cluster yaml, the validation does not read configurations from the yaml context. So if none custom flags are explicitly set in this step, tanzu cli will validate with defualt value

Reply:
updated the wording to make it more clear